lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -218,6 +222,26 @@ class VersionSectionSelector extends Component<SelectorProps> {
       : <span><BsLabel bsStyle={color}>{text}</BsLabel></span>
   }
 
+  // Render the number of transformations applied to the current feed version
+  _renderTransformResultsCount = (version: FeedVersion) => {
+    let totalCount
+    if (version.feedTransformResult) {
+      const {tableTransformResults} = version.feedTransformResult
+      totalCount = tableTransformResults.reduce((prev, cur) => {
+        return cur.deletedCount + cur.updatedCount + cur.addedCount + prev
+      }, 0)
+    } else {
+      totalCount = 0
+    }
+    return (
+      <div style={{display: 'inline'}}>
```

Comment:
Do we need this extra `<div>` (especially since it is set to an inline object)?

Reply:
Nope!